src/app/components/Results/ResultsList.jsx

```diff
@@ -86,9 +94,9 @@ class ResultsList extends React.Component {
           </Link>
         </h2>
         <div className="nypl-results-item-description">
-        <span className="nypl-results-media">{ materialType }</span>
-        <span className="nypl-results-room">{ location }</span>
-        <span className="nypl-results-use">{ usageType }</span>
+          <span className="nypl-results-media">{materialType}</span>
```

Comment:
this will change slightly but we can leave as is for now

https://github.com/NYPL/design-toolkit/issues/139